### PR TITLE
Update drone CI - Test matrix extended for the IOS and JQUERY tests

### PR DESCRIPTION
**Pull request by AntonSermyazhko**

### CODE REVIEW

**Review by San4es — DISMISSED** [review]
LGTM

File: .drone.yml [hunk 1]
```diff
@@ -44,6 +44,11 @@ matrix:
     - { TARGET: test, CONSTEL: ui.grid }
     - { TARGET: test, CONSTEL: ui.scheduler }
     - { TARGET: test, CONSTEL: viz }
+    - { TARGET: test, IOS: true, JQUERY: true, CONSTEL: ui,  }
```
Comment (San4es) [inline .drone.yml]: What about "export" and "misc" constellation?

Comment (AntonSermyazhko) [reply .drone.yml]: These tests are stable on the farm. So, if the problems occurs - we will add them too.

**Review by pavelgruba — DISMISSED** [review]

**Review by roman-simionov — DISMISSED** [review]

**Review by vconst — DISMISSED** [review]

**Review by AlekseyMartynov — DISMISSED** [review]
Please contact me on Teams.

**Review by churkin — DISMISSED** [review]

**Review by jsdmitry — DISMISSED** [review]

**Comment by AlekseyMartynov** [comment]
With the suggested changes, execution never reaches the `elif [ "$IOS" == "true" ]` branch because of

https://github.com/DevExpress/DevExtreme/blob/c9026294554d5e2150c2792954908ac9a420b8f1/.drone.yml#L28-L29

I have discussed this with @AntonSermyazhko. They will prepare a better patch.

**Review by vconst — APPROVED** [review]

**Review by AlekseyMartynov — APPROVED** [review]
LGTM

**Review by San4es — APPROVED** [review]
LGTM

**Pull request merged**